dashboard/src/reducers/datasetListReducer.js

```diff
@@ -1,33 +1,57 @@
-import {
-  GET_PUBLIC_DATASETS,
-  FAVORITED_DATASETS,
-  UPDATE_PUBLIC_DATASETS,
-} from "../actions/types";
+import * as CONSTANTS from "assets/constants/browsingPageConstants";
+import * as TYPES from "actions/types";
 
 const initialState = {
   publicData: [],
   favoriteRepoNames: [],
   tableData: [],
+  offset: CONSTANTS.START_PAGE_NUMBER - 1,
```

Comment:
I apologize for pushing you to use `START_PAGE_NUMBER` here -- that's inappropriate:  the "units" of `START_PAGE_NUMBER` are _pages_ while the units of `offset` are _results_ (so, it's the "wrong zero"...).

The "correct" expression here requires converting the value of `CONSTANTS.START_PAGE_NUMBER - 1` from pages to "results" by multiplying it by `CONSTANTS.DEFAULT_PER_PAGE` and truncating or rounding the result down to the next integer.

However, since we know that the answer is zero, we should probably just use a literal -- that's better than using the wrong constant -- unless you want to add a symbolic constant for the initial result offset (which would be good...there's one other place where we use a literal zero for the results index where we could use it!).

Reply:
Got it